README.md

```diff
@@ -19,13 +19,60 @@ To cope with it, Zed's flatpak defaults can be changed to:
     $ flatpak override --user dev.zed.Zed --env=ZED_FLATPAK_NO_ESCAPE=1
   ```
 
-### Execute commands on the host system
+### Execute commands in the host system
 
 When Zed's flatpak is running in the sandbox with no escape, it is not possible to execute commands on the host system.
 
 To execute commands on the host system, run inside the sandbox:
-```shell
-  $ flatpak-spawn --host <COMMAND>
+
+```bash
+$ flatpak-spawn --host <COMMAND>
+```
+
+or
+
+```bash
+$ host-spawn <COMMAND>
+```
+
+- Most users seem to report a better experience with `host-spawn`
+
+### Use host shell in the integrated terminal.
+
+Another option to execute commands is to use your host shell in the integrated terminal instead of the sandbox one.
+
+For that, open Zed's settings via <kbd>Ctrl</kbd> + <kbd>,</kbd>
+
+And make sure that you have the following lines there:
+
+`flatpak-spawn --host`
+
+```json
+{
+  "terminal": {
+    "shell": {
+      "with_arguments": {
+        "program": "/usr/bin/flatpak-spawn",
+        "args": ["--host", "--env=TERM=xterm-256color", "bash"]
+      }
+    }
+  },
+}
+```
+
+`host-spawn`
+
+```json
+{
+  "terminal": {
+    "shell": {
+      "with_arguments": {
+        "program": "/app/bin/host-spawn",
+        "args": ["bash"]
```

Comment:
There's a lot of different shell out there. Can you make the examples more generic?